core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -42,329 +47,554 @@ object TopicCommand extends Logging {
-                           .withRequiredArg
-                           .describedAs("hosts")
-                           .ofType(classOf[String])
-    val listOpt = parser.accepts("list", "List all available topics.")
-    val createOpt = parser.accepts("create", "Create a new topic.")
-    val deleteOpt = parser.accepts("delete", "Delete a topic")
-    val alterOpt = parser.accepts("alter", "Alter the number of partitions, replica assignment, and/or configuration for the topic.")
-    val describeOpt = parser.accepts("describe", "List details for the given topics.")
-    val topicOpt = parser.accepts("topic", "The topic to create, alter, describe or delete. It also accepts a regular " +
+    private val bootstrapServerOpt = parser.accepts("bootstrap-server", "REQUIRED: The Kafka server to connect to. In case of providing this, a direct Zookeeper connection won't be required.")
+      .withRequiredArg
+      .describedAs("server to connect to")
+      .ofType(classOf[String])
+    private val commandConfigOpt = parser.accepts("command-config", "Property file containing configs to be passed to Admin Client. " +
+      "This is used only with --bootstrap-server option for describing and altering broker configs.")
+      .withRequiredArg
+      .describedAs("command config property file")
+      .ofType(classOf[String])
+    private val zkConnectOpt = parser.accepts("zookeeper", "DEPRECATED, The connection string for the zookeeper connection in the form host:port. " +
+      "Multiple hosts can be given to allow fail-over.")
+      .withRequiredArg
+      .describedAs("hosts")
+      .ofType(classOf[String])
+    private val listOpt = parser.accepts("list", "List all available topics.")
+    private val createOpt = parser.accepts("create", "Create a new topic.")
+    private val deleteOpt = parser.accepts("delete", "Delete a topic")
+    private val alterOpt = parser.accepts("alter", "Alter the number of partitions, replica assignment, and/or configuration for the topic.")
+    private val describeOpt = parser.accepts("describe", "List details for the given topics.")
+    private val topicOpt = parser.accepts("topic", "The topic to create, alter, describe or delete. It also accepts a regular " +
                                            "expression, except for --create option. Put topic name in double quotes and use the '\\' prefix " +
                                            "to escape regular expression symbols; e.g. \"test\\.topic\".")
                          .withRequiredArg
                          .describedAs("topic")
                          .ofType(classOf[String])
-    val nl = System.getProperty("line.separator")
-    val configOpt = parser.accepts("config", "A topic configuration override for the topic being created or altered."  +
+    private val nl = System.getProperty("line.separator")
+    private val configOpt = parser.accepts("config", "A topic configuration override for the topic being created or altered."  +
```

Comment:
Can we update the description stating that update config is only supported with zk option